data/mods/Magiclysm/harvest.json

```
@@ -344,5 +344,33 @@
       { "drop": "sinew", "type": "bone", "mass_ratio": 0.00035 },
       { "drop": "bone", "type": "bone", "mass_ratio": 0.1 }
     ]
+  },
+  {
+    "id": "ogre",
+    "type": "harvest",
+    "entries": [
+      { "drop": "meat", "type": "flesh", "mass_ratio": 0.32 },
+      { "drop": "lung", "type": "flesh", "mass_ratio": 0.0035 },
+      { "drop": "liver", "type": "offal", "mass_ratio": 0.01 },
+      { "drop": "brain", "type": "flesh", "mass_ratio": 0.005 },
+      { "drop": "kidney", "type": "offal", "mass_ratio": 0.002 },
+      { "drop": "raw_leather", "type": "skin", "mass_ratio": 0.02 },
+      { "drop": "sinew", "type": "bone", "mass_ratio": 0.00035 },
+      { "drop": "bone", "type": "bone", "mass_ratio": 0.1 }
```

```suggestion
      { "drop": "brain", "type": "flesh", "mass_ratio": 0.0025 },
      { "drop": "kidney", "type": "offal", "mass_ratio": 0.002 },
      { "drop": "raw_leather", "type": "skin", "mass_ratio": 0.02 },
      { "drop": "sinew", "type": "bone", "mass_ratio": 0.00035 },
      { "drop": "bone", "type": "bone", "mass_ratio": 0.1025 }
```
More bone, less brain.